must-call-checker/src/main/java/org/checkerframework/checker/mustcall/MustCallVisitor.java

```diff
@@ -0,0 +1,76 @@
+package org.checkerframework.checker.mustcall;
+
+import com.sun.source.tree.AnnotationTree;
+import java.util.Collections;
+import java.util.Set;
+import javax.lang.model.element.AnnotationMirror;
+import javax.lang.model.element.ExecutableElement;
+import org.checkerframework.common.basetype.BaseTypeChecker;
+import org.checkerframework.common.basetype.BaseTypeVisitor;
+import org.checkerframework.framework.type.AnnotatedTypeMirror.AnnotatedExecutableType;
+
+/**
+ * The visitor for the MustCall checker. This visitor is similar to BaseTypeVisitor, but overrides
+ * methods that don't work well with the MustCall type hierarchy because it uses the bottom type as
```

Comment:
Would this be solved by having a `@MustCallBottom`?

Reply:
No, because the problem is related to any non-top default. I'll clarify this documentation.